README.md

```diff
@@ -128,33 +128,45 @@ EdgeMesh satisfies the new requirements in edge scenarios (e.g., limited edge re
 
 #### Future Works
 
-<img src="./images/em-intro.png" style="zoom:80%;" />
+![image](./images/em-intro.png)  
 
 At present, the implementation of EdgeMesh relies on the connectivity of the host network. In the future, EdgeMesh will realize the capabilities of CNI plug-ins, and realize the Pod network connectivity between edge nodes and nodes on the cloud, or edge nodes across LANs in a  compatible manner with mainstream CNI plug-ins (e.g., flannel / calico, etc). Finally, EdgeMesh can even replace part of its own components with cloud-native components (e.g., replacing [kube-proxy](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-proxy/) to achieve the capabilities of the Cluster IP, replacing [node local dns cache ](https://kubernetes.io/docs/tasks/administer-cluster/nodelocaldns/) to achieve node-level dns capabilities, and replace [envoy](https://www.envoyproxy.io/) to achieve mesh-layer capabilities).
 
 
 
 ## Architecture
 
-<img src="images/em-arch.png" style="zoom:67%;" />
+![image](./images/em-arch.png)  
 
-To ensure the capability of service discovery in some edge devices with low-version kernels or low-version iptables, EdgeMesh adopts the userspace mode in its implementation of the traffic proxy. In addition, it also comes with a lightweight DNS resolver. As shown in the figure above, the core components of EdgeMesh include:
+To ensure the capability of service discovery in some edge devices with low-version kernels or low-version iptables, EdgeMesh adopts the userspace mode in its implementation of the traffic proxy. In addition, it also comes with a lightweight DNS resolver.
+As shown in the figure above, EdgeMesh contains EdgeMesh-Server microservices and EdgeMesh-Agent microservices.  
```

Comment:
microservices or components? I think component  may be more appropriate

Reply:
EdgeMesh-Server and EdgeMesh-Agent are microservices of EdgeMesh Project, and both of them include some componnets

Reply:
I think component might be better